src/de/map.rs

```diff
@@ -166,12 +178,36 @@ impl<'de, 'a, R: BorrowingReader<'de>> de::MapAccess<'de> for MapAccess<'de, 'a,
             State::Attribute => {
                 let decoder = self.de.reader.decoder();
                 match self.next_attr()? {
-                    Some(a) => seed.deserialize(EscapedDeserializer::new(a.value, decoder, true)),
+                    Some(a) => {
+                        //FIXME: we have to clone value because of wrong lifetimes on `a`
+                        // It should be bound to the input lifetime, but it instead bound
+                        // to a deserializer lifetime
+                        let value: Vec<_> = a.value.into_owned();
+                        seed.deserialize(SimpleTypeDeserializer::new(value.into(), true, decoder))
+                    }
```

Comment:
This is a temporary solution. I'll remove this clone in next PRs, when I figure out how to do that. In any case, struct / map deserialization should be reviewed to match expectations that I documenting in #369